pipelines/BII/BII_pipeline.md

```diff
@@ -0,0 +1,74 @@
+
+## Before you start
+There are no data or API keys required for this analysis. To view the global layer, go to our [STAC catalog](https://stac.geobon.org/viewer/bii_nhm/bii_nhm_10km_2020).
+
+## Running the pipeline
+
+### Pipeline inputs
+The Natural History Museum has created raster layers of BII since the year 2000. BON in a Box has a pipeline to calculate summary statistics and plot a time series from these layers in a country, region, or custom study area of interest. The pipeline has the following inputs:
+
+- **Country:** country of interest.
+
+- **State/Province:** region of interest.
+
+- **Summary statistic:** the user can choose the summary statistic for BII (options: mean, median, mode) that will be calculated for the country or region of interest.
+
+- **Start year for BII raster comparison:** reference BII year for raster plotting.
+
+- **End year for BII comparison:** BII layer to compare to the start year.
+
+- **Coordinate reference system:** the coordinate reference system of choice. Search for a CRS of interest [here](https://epsg.io/). This needs to be a projected coordinate reference system (in meters).
+
+- **Spatial resolution:** the spatial resolution of the rasters (in meters). The spatial resolution of the layer is 10 x 10 km.
+
+### Pipeline steps
+
+#### **1. Retrieving the country bounding box**
+This step retrieves the bounding box for the country/region of interest using the `Get country bounding box` pipeline.
+
+#### **2. Loading data from the GEO BON STAC catalog**
+This step extracts the global biodiversity intactness layers from various collections on the GEO BON Spatio Temporal Asset Catalog. The layers are in EPSG: 4326 and 10x10 km resolution but the user can specify other coordinate references systems and spatial resolutions.
+
+#### **3. Calculating zonal statistics**
+This step calculates the zonal statistics for the raster layers obtained from the GEO BON STAC catalog over the bounding box from step 1, using the R package exactextractr. The user can choose what types of zonal statistics will be extracted.
+
+#### **4. Calculating the BII change**
+This step generates a raster of the change in the BII between the two chosen time points. The BII uses the PREDICTS database to establish a reference state using the biodiversity patterns in habitats with minimal disturbance levels. Then, it assigns sensitivity scores to each species based on their vulnerability to human pressure. Intactness is calculated by comparing the observed species abundance in a given area to what is expected under reference conditions with low human impact.
```

Comment:
Only the first sentence is really about the step. The other one is the generic description of how to calculate BII. This wold better fit step 2, where we can say: load the data from the STAC, and by the way this is how it was made.